setup.py

```diff
@@ -21,7 +21,7 @@
     },
     entry_points={
         'console_scripts': [
-            'ocrd-kwalitee=kwalitee.cli:cli',
+            'ocrd-quiver=quiver.cli:cli',
```

Comment:
Can we find a name that does not start with `ocrd-`? Like `quiver-ocrd` or `ocrd_quiver`? Otherwise core will waste a few cycles of CPU and I/O if `ocrd-quiver` is in the path, when looking up processor-specific resources.

Reply:
Ah, I didn't think about that. Then I'll just go for `quiver`, AFAICT this doesn't interfere with any popular Python package.

Reply:
I thought about this a little more and will settle for `quiver-ocrd`.